libcxx/test/std/utilities/template.bitset/bitset.members/left_shift_eq.pass.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+// ADDITIONAL_COMPILE_FLAGS(has-fconstexpr-steps): -fconstexpr-steps=15000000
 // bitset<N>& operator<<=(size_t pos); // constexpr since C++23
```

Comment:
```suggestion
// ADDITIONAL_COMPILE_FLAGS(has-fconstexpr-steps): -fconstexpr-steps=15000000

// bitset<N>& operator<<=(size_t pos); // constexpr since C++23
```